src/app/scripts/cac/control/cac-control-directions-list.js

```diff
@@ -93,6 +93,18 @@ CAC.Control.DirectionsList = (function (_, $, MapTemplates, Social, UserPreferen
 
         $container.empty();
 
+        // TODO: remove this SEPTA rail warning when contingency plan ends and/or update
+        // when modified rail schedule published.
+
+        // Show alert regarding current SEPTA rail status
+        _.every(itinerary.legs, function(leg) {
+            if (leg.transitLeg && leg.mode === 'RAIL' && leg.agencyName === 'SEPTA') {
+                $container.append(MapTemplates.septaRailWarningAlert());
+                return false; // break on first SEPTA rail leg found
+            }
+            return true;
+        });
```

Comment:
In the pantheon of very similar lodash functions, `some` might be more straightforward here, e.g.:
```
if(_.some(itinerary.legs, function(leg) {
    return leg.transitLeg && leg.mode === 'RAIL' && leg.agencyName === 'SEPTA'; })) {
    $container.append(MapTemplates.septaRailWarningAlert());
}
```
Thought the spacing on that comes out a little crunchy.

Either way works fine.